client/checkout/blocks/payment-method-label.js

```diff
@@ -12,9 +12,50 @@ import './style.scss';
 import { useEffect, useState } from '@wordpress/element';
 import { getAppearance } from 'wcpay/checkout/upe-styles';
 
-export default ( { api, upeConfig, upeName, upeAppearanceTheme } ) => {
+const bnplMethods = [ 'affirm', 'afterpay_clearpay', 'klarna' ];
+const PaymentMethodMessageWrapper = ( {
```

Comment:
Should this be on its own file?

Reply:
I personally don't think so, because it's not going to be used anywhere else - I just extracted it from the previous implementation because I thought it would make things slightly easier to scroll through, since there are a lot of `if` conditions to check for. 🤷